src/components/Modal/ConnectWalletModal/index.tsx

```diff
@@ -103,6 +114,34 @@ function ConnectWalletModal(props: ReactModal.Props) {
           } as WalletButtonProps,
         ]
       : []),
+    ...wm.wallets.map(
+      ({ info: { logo, windowKey, name, prettyName, downloads } }) => {
+        const isInstalled =
+          windowKey &&
+          (window as unknown as Record<string, unknown>)[windowKey];
```

Comment:
```suggestion
        const isInstalled =
          !!(windowKey &&
          (window as unknown as Record<string, unknown>)[windowKey]);
```

should this be evaluated to boolean?

Reply:
[4d6daa4](https://github.com/dezswap/dezswap-web-app/pull/239/commits/4d6daa4174e7802bb037bb6e4d406e7bb52018ae) is updated